policies/sample_paloalto.pol

```diff
@@ -0,0 +1,68 @@
+################
+### RULEBASE ###
+################
+
+# NOTE: makes sure zone names correspond to those specified in your firewall setup
+
+header {
+  target:: paloalto from-zone Trust-L3 to-zone Trust-L3
+}
+
+term allow-ping-firewall{
+  destination-address:: FIREWALL_EXTERNAL
```

Comment:
FIREWALL_EXTERNAL doesn't exist in our .net files. Can you please fix this? Either add something into multitarget_sample.pol (I did this to test some of your code) or fix this file to use things already in the .net files.